docs/docs/get_around_unsupported_features.md

```diff
@@ -0,0 +1,25 @@
+# Get around unsupported features.
+
+There is no algorithm to get around unsupported features, in most scenarios is needed to understand what is the piece of code doing. There are some unsupported features that are impossible to rewrite right now, understanding what is expected to do when translated to Cairo may indicate that the logic can just be removed.
```

Comment:
Try changing these parts to:
- in most scenarios **it** is necessary to understand what the piece of code **is** doing
- understanding what **it** is expected to do when translated to Cairo